conf/machine/include/imx-base.inc

```diff
@@ -267,9 +267,9 @@ PREFERRED_VERSION_libdrm_mx7 ?= "2.4.91.imx"
 PREFERRED_VERSION_libdrm_mx8 ?= "2.4.91.imx"
 
 # Use i.MX optee Version
-PREFERRED_VERSION_optee-os_mx8     ?= "3.2.0.imx"
-PREFERRED_VERSION_optee-client_mx8 ?= "3.2.0.imx"
-PREFERRED_VERSION_optee-test_mx8   ?= "3.2.0.imx"
+PREFERRED_VERSION_optee-os     ?= "3.2.0.imx"
+PREFERRED_VERSION_optee-client ?= "3.2.0.imx"
+PREFERRED_VERSION_optee-test   ?= "3.2.0.imx"
```

Comment:
This does not seems well tested.

Reply:
What do you mean?

Reply:
We should not carry the two versions. We should upgrade it.

So this development should be finished and send here when it is considered good for general use.

Why we cannot move master to this?

Reply:
Optee 3.7.0 should be released very soon but not ready yet.
I'm okay with the fact you don't want two version of Optee.
I will come back when it's ready and do the full upgrade instead of having two version.
Thanks!